pkg/api/v1/types.go

```diff
@@ -105,6 +106,7 @@ func init() {
 
 // VirtualMachine is *the* VM Definition. It represents a virtual machine in the runtime environment of kubernetes.
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +k8s:openapi-gen=true
```

Comment:
If you change this to:

```golang
// ---
// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
//  +k8s:openapi-gen=true
```

then the doc generator will not pick up everything after `---`.
We should have done the same for +optional and +deepcopy too ...

Reply:
oh, excellent, I'll make this change. That's what I was looking for